lib/session.js

```diff
@@ -203,7 +203,7 @@ Session.prototype.createLink = function(linkPolicy) {
   var self = this;
   link.on(Link.Detached, function(details) {
     debug('detached(' + link.name + '): ' + (details ? details.error : 'No details'));
-    if (!link.shouldReattach() || self.disposed) self._removeLink(link);
+    if (!link.shouldReattach() || self._disposed) self._removeLink(link);
```

Comment:
Were we calling the wrong property on the object before?

Reply:
yeah that's my bad...

Reply:
actually wait the naming is fine. i defined a getter on the class called `disposed` which is the value of `this._disposed` so this should be fine

Reply:
see line 156

Reply:
agree - this is more of a nitpicking thing / just for consistency purposes.